doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -475,3 +475,17 @@ docker tag docker.io/busybox:1.28.4 myopenshiftcluster/openshift/busybox:latest
 +
 ** For  {OPENSHIFT} version 3.11, see the "Accessing the Registry" section of the https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html-single/developer_guide/index[OpenShift Container Platform 3.11 Developer Guide].
 ** For  {OPENSHIFT} version 4.1, see the "Accessing the Registry" section of https://access.redhat.com/documentation/en-us/openshift_container_platform/4.1/html-single/registry/index[Configuring registries for OpenShift Container Platform 4.1].
+
+== Offline Maven repository
+
+.The `offliner` tool reports errors when downloading artifacts for an offline Maven repository [https://issues.redhat.com/browse/RHPAM-2234[RHPAM-2234]]
+
+Issue: When you run the `offliner` tool to download artifacts for an offline Maven repository, the tool reports that downloading some artifacts has failed. The error message is `The artifact was not found in any of the provided repositories`.
```

Comment:
I would say:
the tool reports that some artifacts failed to download.

But if you think you need SME approval for the change, leave it.